FuzzoBot/Program.cs

```diff
@@ -88,4 +91,4 @@ private static ServiceProvider ConfigureServices( /*IConfiguration configuration
             .AddSingleton<MessageReceivedHandler>()
             .BuildServiceProvider();
     }
-}
\ No newline at end of file
+}
```

Comment:
hmm, what's this?

Reply:
Ah, it seems that there was a newline added at the end of the file automatically because of Vim autoformatting?